.github/workflows/build-and-test-types.yml

```diff
@@ -115,7 +111,7 @@ jobs:
 
       # Note: We currently expect "FalseCJS" failures for Node16 + `moduleResolution: "node16",
       - name: Run are-the-types-wrong
-        run: npx @arethetypeswrong/cli ./package.tgz --format table --ignore-rules false-cjs
+        run: npx @arethetypeswrong/cli@latest ./package.tgz --format table --ignore-rules false-esm
```

Comment:
:question: why the rule change?

Reply:
Now we have reselect.d.mts because of tsup version 8.